framework/aws-kms/aws-kms-hierarchical-keyring.md

```diff
@@ -293,18 +293,18 @@ When the hierarchical keyring receives an OnEncrypt request,
 the cache entry identifier MUST be calculated as the first 32 bytes of the
 SHA-512 hash of the following byte strings, in the order listed:
```

Comment:
imo; it would be best to split these up similar to how we write the requirements of other structures or orders throughout the spec since it can be difficult to add duvet annotations to these charts.
For example,
```suggestion
the cache entry identifier MUST be calculated as the first 32 bytes of the
SHA-512 hash of the following byte strings, in the order listed:
- MUST be the length of the branch key id in its uint8 interpreatation
- MUST be the SHA512 output of the UTF8 encoded branch-key-id
- MUST be a single null byte `0x00`
- MUST be the constant UTF8 encoded string "ACTIVE" 
```

This way we can easily duvet the implementation.

Reply:
Sounds good. Thanks for pointing this out.